commit 1a8364456c2f3946b4feb8fc78eaf00d974f4c03
Author: Jan Kiszka <jan.kiszka@siemens.com>
Date:   Wed Feb 23 09:28:53 2011 +0100

    qemu-kvm: Fix non-PCI target build
    
    Replace obsolete qemu-kvm.h with kvm.h in pci.c and build that module
    just like upstream does. This fixes non-x86 targets which have no PCI
    support.
    
    Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
    Signed-off-by: Avi Kivity <avi@redhat.com>

diff --git a/Makefile.objs b/Makefile.objs
index f5702eb..3ec7121 100644
--- a/Makefile.objs
+++ b/Makefile.objs
@@ -170,7 +170,7 @@ hw-obj-y =
 hw-obj-y += loader.o
 hw-obj-$(CONFIG_VIRTIO) += virtio.o virtio-console.o
 hw-obj-y += fw_cfg.o
-hw-obj-$(CONFIG_PCI) += pci_bridge.o
+hw-obj-$(CONFIG_PCI) += pci.o pci_bridge.o
 hw-obj-$(CONFIG_PCI) += msix.o msi.o
 hw-obj-$(CONFIG_PCI) += pci_host.o pcie_host.o
 hw-obj-$(CONFIG_PCI) += ioh3420.o xio3130_upstream.o xio3130_downstream.o
diff --git a/Makefile.target b/Makefile.target
index 6e9a024..23367eb 100644
--- a/Makefile.target
+++ b/Makefile.target
@@ -195,7 +195,7 @@ endif #CONFIG_BSD_USER
 # System emulator target
 ifdef CONFIG_SOFTMMU

-obj-y = arch_init.o cpus.o monitor.o pci.o machine.o gdbstub.o vl.o balloon.o
+obj-y = arch_init.o cpus.o monitor.o machine.o gdbstub.o vl.o balloon.o
 # virtio has to be here due to weird dependency between PCI and virtio-net.
 # need to fix this properly
 obj-$(CONFIG_NO_PCI) += pci-stub.o
diff --git a/hw/pci.c b/hw/pci.c
index 0c44939..1f6cebe 100644
--- a/hw/pci.c
+++ b/hw/pci.c
@@ -29,8 +29,8 @@
 #include "net.h"
 #include "sysemu.h"
 #include "loader.h"
-#include "qemu-kvm.h"
 #include "hw/pc.h"
+#include "kvm.h"
 #include "device-assignment.h"
 #include "qemu-objects.h"
 #include "range.h"
